modules/HardwareDrivers/PowerMeters/DZG_GSH01/app_layer.hpp

```diff
@@ -0,0 +1,357 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2025 Pionix GmbH and Contributors to EVerest
+
+/*
+ This is an implementation for the GSH01 powermeter application layer
+*/
+#ifndef APP_LAYER
+#define APP_LAYER
+
+#include "ld-ev.hpp"
+#include <chrono>
+#include <everest/crc/crc.hpp>
+#include <everest/logging.hpp>
+#include <generated/types/powermeter.hpp>
+#include <optional>
+#include <stdint.h>
```

Comment:
This could make us of `<cstdint>` as well (would need `std::` prefixes before all (u)int_xyz usage though
```suggestion
#include <cstdint>
```